src/App.css

```diff
@@ -342,4 +342,7 @@ td, tr{
 }
 .inputs input::placeholder,textarea{
   color: #917f7f!important;
+}
+.imgNoFound{
+  transform: translateX(60%);
```

Comment:
I believe you meant to use the transform here to center the image, but this is not a good approach.
You can  use something like 
display :flex 
justify-content : center 

to make the image is centered.

one thing else that you need to do is to make the image container outside of the table, because making it at the table will make the image container takes one column.